src/main/java/com/educare/unitylend/dao/UserRepository.java

```diff
@@ -13,22 +10,34 @@
 @Repository
 public interface UserRepository {
 
-    @Select(SELECT_USERS)
-    List<User> getAllUsers();
-
-    @SelectProvider(type = UserRepository.class, method = "getUserForUserIdQuery")
-    User getUserForUserId(@Param("userId") String userId);
-
-    static final String SELECT_USERS = "select user_id as userID, password, name from user_table";
-
     static String getUserForUserIdQuery(String userId){
         StringBuilder sqlQueryBuilder = new StringBuilder();
 
-        sqlQueryBuilder.append("select user_id as userID, password, name from user_table where 1=1 ");
+        sqlQueryBuilder.append("select userid as userID, password, name,email,dob,income,officename,collegeuniversity,locality from User where 1=1 ");
         if(userId != null){
             sqlQueryBuilder.append(" and userId = %s".formatted(userId));
         }
 
         return sqlQueryBuilder.toString();
     }
+
+    @Select("select * from tempuser")
+    List<User> getAllUsers();
+
+    @SelectProvider(type = UserRepository.class, method = "getUserForUserIdQuery")
+    User getUserForUserId(@Param("UserId") String userId);
+    //  static final List<String> SELECT_USERS = "select password from User";
+
+    @Insert("INSERT INTO tempuser (userid, password, name,email,dob,income,officename,collegeuniversity,locality) VALUES (CAST(uuid_generate_v4() AS VARCHAR), #{password}, #{name}, #{email},#{dob},#{income},#{officename},#{collegeuniversity},#{locality})")
+    void createUser(User user);
+
+
+    @Select("SELECT userid FROM tempuser WHERE email = #{email}")
+    String settingID(@Param("email") String email);
+
+
+    @Update("UPDATE tempuser SET name = #{name}, email =#{email}, locality = #{locality}, officename = #{officename},collegeuniversity=#{collegeuniversity} income = #{income} WHERE userid = #{userid}")
```

Comment:
Its better to define complex sql queries separately from the function definition to improve the readability

Reply:
It has been done.